src/uwtools/tests/config/test_support.py

```diff
@@ -67,7 +68,7 @@ def test_log_and_error(caplog):
 
 
 def test_represent_namelist():
-    namelist = reads("&namelist\n key = value\n/\n")
+    namelist = f90nml.reads("&namelist\n key = value\n/\n")
```

Comment:
I think `Namelist` says enough on its own not to be `f90nml.Namelist`, but `reads` seemed generic enough to warrant an explicit namespace for clarity.

Reply:
Thats fair, no complaints!